buf/registry/policy/v1beta1/configuration.proto

```diff
@@ -0,0 +1,144 @@
+  message LintConfig {
+    // The check config.
+    CheckConfig check_config = 1;
+    // The suffix that controls the behavior of the ENUM_ZERO_VALUE_SUFFIX lint rule. By default,
+    // this rule verifies that the zero value of all enums ends in `_UNSPECIFIED`, however, this
+    // allows organizations to choose a different suffix.
+    string enum_zero_value_suffix = 2 [(buf.validate.field).string.max_len = 250];
+    // This bool allows the same message type to be used for a single RPC's request and response type.
+    bool rpc_allow_same_request_response = 3;
+    // This bool allows RPC requests to be google.protobuf.Empty messages.
+    bool rpc_allow_google_protobuf_empty_requests = 4;
+    // This bool allows RPC responses to be google.protobuf.Empty messages.
+    bool rpc_allow_google_protobuf_empty_responses = 5;
+    // This suffix controls the behavior of the SERVICE_SUFFIX lint rule. By default, this rule
+    // verifies that all service names are suffixed with `Service`, however this allows organizations
+    // to choose a different suffix.
+    string service_suffix = 6 [(buf.validate.field).string.max_len = 250];
+  }
+  // A breaking config consists of the generic check config and additional breaking-specifc configs
+  // required for running breaking change detection.
+  message BreakingConfig {
+    // The check config.
+    CheckConfig check_cofnig = 1;
+    // This bool determines whether to ignore unstable packages for breaking change detection:
+    //   - v\d+test.*
+    //   - v\d+(alpha|beta)\d*
+    //   - v\d+p\d+(alpha|beta)\d*
+    bool ignore_unstable_packages = 2;
+  }
+  // A plugin config consists of the configs for invoking a check plugin.
+  message CheckPluginConfig {
+    // The fully-qualified name of a check Plugin within a BSR instance.
+    //
+    // A Name uniquely identifies a Plugin.
+    message Name {
+      // The name of the Organization owner.
+      string owner = 1 [
+        (buf.validate.field).required = true,
+        (buf.validate.field).string.max_len = 32
+      ];
+      // The name of the Plugin.
+      string plugin = 2 [(buf.validate.field).string = {
+        min_len: 2
+        max_len: 100
+      }];
+    }
+    // A Plugin option. This consists of a key and value.
+    message Option {
+      string key = 1 [(buf.validate.field).string = {
+        min_len: 2
+        max_len: 100
+      }];
+      string value = 2 [(buf.validate.field).string = {
+        min_len: 2
+        max_len: 100
+      }];
+    }
+    // The Plugin name.
+    Name name = 1 [(buf.validate.field).required = true];
+    // The ref to a specific Plugin commit. This could be a label name or a commit ID.
+    // If no reference is set, then at the time of running the Policy, the latest commit on
+    // the default label of the Plugin is used.
+    //
+    // If this value is present but empty, this should be treated as not present.
+    string ref = 2 [(buf.validate.field).string.max_len = 250];
```

Comment:
Should we allow users to leave the `ref` unspecified, e.g.

```
plugins:
  - plugin: buf.build/bufbuild/timestamp-suffix
```

This would mean that each time this policy is run, it would use the latest commit on the default label for the plugin.

Discussing this with @mfridman, this is similar to specifying `runs-on: ubuntu-latest` for github actions runners -- for each run of the action, the user can introspect what `ubuntu-latest` is and they would get all updates/changes to `ubuntu-latest` without having to change their job configuration. This feels like a desirable UX.

Reply:
Resolved this in our last discussion, we will be allowing plugin name. This will not be captured here, since we are no longer putting this configuration into proto.